tutorials/terraform/data-proc-data-exchange-with-mch.tf

```diff
@@ -0,0 +1,288 @@
+# Infrastructure for Yandex Cloud Managed Service for ClickHouse cluster, Data Proc cluster and virtual machine.
+#
+# RU: https://cloud.yandex.ru/docs/data-proc/tutorials/exchange-data-with-mch
+#
+# Set the configuration of the Managed Service for ClickHouse cluster, Data Proc cluster and Virtual Machine.
```

Comment:
Set the configuration of the Managed Service for ClickHouse cluster, Data Proc cluster, and Virtual Machine.